apps/src/lib/kits/maker/CircuitPlaygroundBoard.js

```diff
@@ -67,7 +67,7 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
         this.fiveBoard_ = board;
         resolve();
       });
-      board.once('error', reject);
+      board.on('error', reject);
```

Comment:
Weird edge case: When this timeout occurs, johnny-five actually emits two error events (probably not on purpose?) - _and_ the event emitter throws if an 'error' event is raised but no error handler is attached.  Here it was throwing inside a `setTimeout` because we detached our error handler after the first event.